services/venice-server/src/test/java/com/linkedin/venice/stats/AggServerHttpRequestStatsTest.java

```diff
@@ -13,10 +13,11 @@
 
 
 public class AggServerHttpRequestStatsTest {
-  protected MetricsRepository metricsRepository;
-  protected MockTehutiReporter reporter;
-  protected AggServerHttpRequestStats singleGetStats;
-  protected AggServerHttpRequestStats batchGetStats;
+  private MetricsRepository metricsRepository, metricsRepositoryForKVProfiling;
+  private MockTehutiReporter reporter, reporterForKVProfiling;
```

Comment:
Can we  follow style consistent with other variable declarations in this class? (one declaration per line)

Reply:
Sure